.github/workflows/rust.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     needs: build
     strategy:
       matrix:
-        version: [ 21.6.0, 20.10.5 ]
+        version: [ 20.10.5-buster-slim, 21.10.0-buster-slim, ci ]
```

Comment:
**CONCERN:** Probably doesn't change much but I think that we initially agreed to use `focal`.

Reply:
Done